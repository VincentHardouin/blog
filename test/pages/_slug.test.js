import { shallowMount } from '@vue/test-utils';
import Page from '@/pages/_slug';

describe('pages | _slug', () => {
  it('should render blog post', () => {
    const wrapper = shallowMount(Page, {
      data() {
        return {
          page: {
            title: 'Article title',
            content: 'Content',
            date: '2021-10-28',
          },
        };
      },
      computed: {
        content() {
          return 'content';
        },
      },
    });

    expect(wrapper.element).toMatchSnapshot();
  });
});
