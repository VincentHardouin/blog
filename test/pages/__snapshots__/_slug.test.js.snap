// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages | _slug should render blog post 1`] = `
<div
  class="page"
>
  <h1
    class="page__title"
  >
    Article title
  </h1>
   
  <div
    class="page__content"
  >
    content
  </div>
</div>
`;
